src/phrases/phrases-en.js

```diff
@@ -887,7 +889,7 @@ const phrases = {
     related: {
       title: 'Related content',
       subtitle: 'Drag article to change the order',
-      showMore: 'Show more related content',
+      showMore: 'Show more related articles',
```

Comment:
Revert denne. Det skal være content og ikkje articles.
```suggestion
      showMore: 'Show more related content',
```